machine/topology_test.go

```diff
@@ -81,98 +135,82 @@ func TestTopology(t *testing.T) {
 	}
 }
 
-func TestTopologyWithSimpleCpuinfo(t *testing.T) {
-	if isSystemZ() {
-		t.Skip("systemZ has no topology info")
-	}
-	sysFs := &fakesysfs.FakeSysFs{}
-	c := sysfs.CacheInfo{
-		Size:  32 * 1024,
-		Type:  "unified",
-		Level: 1,
-		Cpus:  1,
-	}
-	sysFs.SetCacheInfo(c)
-	topology, numCores, err := GetTopology(sysFs, "processor\t: 0\n")
-	if err != nil {
-		t.Errorf("Expected cpuinfo with no topology data to succeed.")
-	}
-	node := info.Node{Id: 0}
-	core := info.Core{Id: 0}
-	core.Threads = append(core.Threads, 0)
-	cache := info.Cache{
-		Size:  32 * 1024,
-		Type:  "unified",
-		Level: 1,
-	}
-	core.Caches = append(core.Caches, cache)
-	node.Cores = append(node.Cores, core)
-	// Copy over Memory from result. TODO(rjnagal): Use memory from fake.
-	node.Memory = topology[0].Memory
-	// Copy over HugePagesInfo from result. TODO(ohsewon): Use HugePagesInfo from fake.
-	node.HugePages = topology[0].HugePages
-	expected := []info.Node{node}
-	if !reflect.DeepEqual(topology, expected) {
-		t.Errorf("Expected topology %+v, got %+v", expected, topology)
-	}
-	if numCores != 1 {
-		t.Errorf("Expected 1 core, found %d", numCores)
-	}
-}
-
-func TestTopologyEmptyCpuinfo(t *testing.T) {
-	if isSystemZ() {
-		t.Skip("systemZ has no topology info")
-	}
-	_, _, err := GetTopology(&fakesysfs.FakeSysFs{}, "")
+func TestTopologyEmptySysFs(t *testing.T) {
+	_, _, err := GetTopology(&fakesysfs.FakeSysFs{})
 	if err == nil {
-		t.Errorf("Expected empty cpuinfo to fail.")
+		t.Errorf("Expected empty sysfs to fail.")
```

Comment:
Ditto.

Reply:
Along with this one.

Reply:
This part of code was earlier changed, it seems to be an issue with github refreshing or presenting PR changes....